Add RT meeting host playbook to the RT Lead handbook (#1973)
kubernetes/sig-release: release-team/role-handbooks/release-team-lead/README.md

- Help fill any open positions for the next release milestone

---

## Release Team Meeting Host Playbook

To coordinate and discuss each release the release lead with support of the shadows are hosting release team meetings (see the section X above).
The following playbook can be used for reference to structure the meeting, you don't need to stick step by step (word by word), it should give guidance how a meeting usually is structured. 

### Requirements to host the meeting as release team release lead / lead shadow
1. Install Zoom and check your settings (especially the audio)
2. Make sure to get the **host key**, otherwise you wont be able to claim host share the screen and start the recording

### Pre-Meeting
1. Announce the release team meeting about an hour before in the sig-release slack channel.

> 

Reviewer: make a new copy in meeting notes and edit details accordingly

```txt
Hi folks! The release meeting takes place 1 hour from now
Links:
* Agenda & Minutes: https://bit.ly/k8s125-releasemtg
* Zoom Link: https://zoom.us/j/406805785?pwd=elQ1aVc0THZ, passcode 77777

If anyone has anything they’d like to discuss, please add to the notes.
```

### Pre-Recording
1. *Slack reminder (thread)* **"The meeting start now!"**
2. *join the meeting*
3. *turn on the camera*
4. *say hello to everyone* **"Hello all, let's wait a few minutes for everyone to join"**
5. *post the meeting notes to the meeting chat* 

> 6. *claim host* (you can claim host by opening the attendees window and clicking claim host)

Reviewer: Insert host key

***wait until 3 minutes past***

7. *post the meeting notes link to the meeting chat* **"I posted the meeting notes link to chat, please add yourself to the attendees list, thank you!"**
8. **"Since it is now 3 minutes past, we will get started"**
9. **"If you rather not get recorded, you can now turn off your video"**
10. *open meeting notes and share screen* (NOTE: on Linux you might need to share the entire screen, a single window does not work sometimes)

### During-Recording
1. *start recording* record to the cloud, the automation will 'automagically' upload the recording to YouTube.
2. *say hello to everyone* **"Hello everyone, my name is 'Alice' and I will be the host for this Kubernetes 'X.XX' release team meeting. Today is 'May' the '4th' and we are in week 'X' of the release cycle.**

> 3. *code of conduct* **"Please be aware that this is a Kubernetes community meeting which falls under the Kubernetes code of conduce, which can be summarized to be excellent to each other, thank you all!"** 

Reviewer: *conduct typo :)

- Help fill any open positions for the next release milestone

---

## Release Team Meeting Host Playbook

> To coordinate and discuss each release the release lead with support of the shadows are hosting release team meetings (see the section X above).

Reviewer 2: ```suggestion
To coordinate and discuss each release the release lead with support of the shadows host release team meetings (see the section X above).
```

The following playbook can be used for reference to structure the meeting, you don't need to stick step by step (word by word), it should give guidance how a meeting usually is structured. 

> ### Requirements to host the meeting as release team release lead / lead shadow
> 1. Install Zoom and check your settings (especially the audio)
> 2. Make sure to get the **host key**, otherwise you wont be able to claim host share the screen and start the recording

Reviewer 2: ```suggestion
### Requirements to host the meeting as Release Team Lead / Lead Shadow
1. Install Zoom and check your settings (especially the audio)
2. Make sure to get the **host key** from the Emeritus Adviser or Release Lead , otherwise you wont be able to claim host share the screen and start the recording
```

```

### Pre-Recording
1. *Slack reminder (thread)* **"The meeting start now!"**
2. *join the meeting*
3. *turn on the camera*
4. *say hello to everyone* **"Hello all, let's wait a few minutes for everyone to join"**
5. *post the meeting notes to the meeting chat* 
6. *claim host* (you can claim host by opening the attendees window and clicking claim host and inserting the host key)

***wait until 3 minutes past***

7. *post the meeting notes link to the meeting chat* **"I posted the meeting notes link to chat, please add yourself to the attendees list, thank you!"**
8. **"Since it is now 3 minutes past, we will get started"**
9. **"If you rather not get recorded, you can now turn off your video"**
10. *open meeting notes and share screen* (NOTE: on Linux you might need to share the entire screen, a single window does not work sometimes)

### During-Recording
1. *start recording* record to the cloud, the automation will 'automagically' upload the recording to YouTube.
2. *say hello to everyone* **"Hello everyone, my name is 'Alice' and I will be the host for this Kubernetes 'X.XX' release team meeting. Today is 'May' the '4th' and we are in week 'X' of the release cycle.**

> 3. *code of conduct* **"Please be aware that this is a Kubernetes community meeting which falls under the Kubernetes code of conduct, which can be summarized to be excellent to each other, thank you all!"** 
> 4. *open discussion items*: If something is on the list, you can read it out loud and wait for the person to respond. If there is nothing on the open discussion list, you can say something like: **"There is nothing in the open discussion section in the meeting notes. Is there anything to discuss before we get into the team updates?"** wait a few seconds - **"Okay, let's move on"** 
> 5. *team updates*: **"Lets start with the team updates"** Call the person giving the update and wait for the update. If no one from the team gives the update, then you can review the last update given in a previous meeting, read it aloud, and then move to the next team. 

Reviewer 2: In the past, I've stated the meeting recording warning on record after the code of conduct but everyone has their own style
```suggestion
3. *code of conduct* **"Please be aware that this is a Kubernetes community meeting which falls under the Kubernetes code of conduct, which can be summarized to be excellent to each other, thank you all!"** 
4. *recording warning* **"This meeting is recorded and will be publicly posted to YouTube so please be mindful what you say is being recorded."**
5. *open discussion items*: If something is on the list, you can read it out loud and wait for the person to respond. If there is nothing on the open discussion list, you can say something like: **"There is nothing in the open discussion section in the meeting notes. Is there anything to discuss before we get into the team updates?"** wait a few seconds - **"Okay, let's move on"** 
6. *team updates*: **"Lets start with the team updates"** Call the person giving the update and wait for the update. If no one from the team gives the update, then you can review the last update given in a previous meeting, read it aloud, and then move to the next team. 
```

- Help fill any open positions for the next release milestone

---

## Release Team Meeting Host Playbook

> Release Team meetings are held to coordinate and discuss release-related activities. The Release Lead and Lead Shadows host Release Team meetings (see the section X above) throughout the release cycle.

Reviewer: update the `see section X above`
